operator/pkg/common/extensions/reconcile_scanner_v4_db_password.go

```diff
@@ -28,7 +28,7 @@ func ReconcileScannerV4DBPassword(ctx context.Context, obj ScannerV4BearingCusto
 
 func reconcileScannerV4DBPassword(ctx context.Context, obj ScannerV4BearingCustomResource, client ctrlClient.Client) error {
 	run := &reconcileScannerV4DBPasswordExtensionRun{
-		SecretReconciliator:  NewSecretReconciliator(client, obj),
+		SecretReconciliator:  NewSecretReconciliator(client, obj, OwnershipStrategyOwnerReference),
```

Comment:
Same comment as for scanner-db.

Reply:
Added a comment to explain why the OwnerReference strategy is used